bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/speaker.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dirigera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="speaker">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Speaker</label>
+		<description>Speaker with player activities</description>
+
+		<channels>
+			<channel id="player" typeId="player">
+				<label>Player</label>
+				<description>Player Control</description>
+			</channel>
+			<channel id="volume" typeId="dimmer">
+				<label>Volume Control</label>
+				<description>Handle volume in percent</description>
+			</channel>
+			<channel id="mute" typeId="switch">
+				<label>Mute Control</label>
```

Comment:
Mute has a system channel, prefer that you use that one:
https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types
Might also applies to other `Things`

Reply:
Done